ouroboros-consensus/test/infra-test/Test/Ouroboros/Consensus/ChainGenerator/Tests/Sync.hs

```diff
@@ -425,23 +463,7 @@ mkChainDb tracer candidateVars nodeCfg registry = do
         (\_ -> ChainDB.Impl.openDBInternal chainDbArgs False)
         (ChainDB.closeDB . fst)
     _ <- forkLinkedThread registry "AddBlockRunner" intAddBlockRunner
-    void $ flip Map.traverseWithKey candidateVars $ \ pid varCandidate ->
-      forkLinkedThread registry (condense pid) $ monitorCandidate chainDB varCandidate
     pure chainDB
-  where
-    monitorCandidate chainDB varCandidate =
```

Comment:
This was the simplified mock of what is now done by `Ouroboros.Network.BlockFetch.Client.blockFetchClient`?

Reply:
Yes, this was the exchanging of blocks without block fetch.